user_sync/config.py

```diff
@@ -83,7 +95,7 @@ def get_logging_config(self):
 
     def get_dashboard_options_for_owning(self):
         owning_config_filename = DEFAULT_DASHBOARD_OWNING_CONFIG_FILENAME
-        owning_config_path = self.get_file_path(owning_config_filename)
+        owning_config_path = self.get_absolute_file_path(owning_config_filename)
```

Comment:
No, this has to be processed as if it were relative to the main config file.  All of the predefined default values have to be relativized as if they were part of the main config file.  At the time you find the main config file and process it with load-yaml, you will need to check to see if the user actually provided values for these constants in the config file.  If not, you will need to compute the right absolute pathnames for them (same directory as the config file but their value as the filename part) and then insert those keys into the result of load-yaml on the main config.  This is why I was recommending you leave the ConfigFileLoader class in place, because it still needs two entries: ConfigFileLoader.load_root_config which does the additional processing I'm describing here, and ConfigFileLoader.load_sub_config which only does what load_yaml does.  But if you want to get rid of ConfigFileLoader that's fine, you can just have two functions in the ConfigLoader class instead.